src/components/schemas/fields.yaml

```diff
@@ -234,6 +234,104 @@ policy_operator:
     Enum value signifying the operator for the field
   example: IN
 
+policy_rule_minimal:
+  type: object
+  description: |
+    Policy rule minimal object
+  properties:
+    id:
+      type: string
+      example: tpr1234abcd
+      description: This represents the id of the policy.
+    description:
+      allOf:
+        - $ref: '#/description'
+      description: |
+        This represents the description of the policy.
+    action_show_warning:
+      type: boolean
+      example: True
+      
+expense_policy_violating_expenses_query_params:
+  type: object
+  description: |
+    Query params for fetching the expenses
+  properties:
+    start_date:
+      type: string
+      example: "2021-12-01"
+    end_date:
+      type: string
+      example: "2021-12-01"
+    state:
+      type: string
+      example: "neq.DRAFT"
+    category_id:
+      type: string
+      example: "in.(60506)"
+    project_id:
+      type: string
+      example: "in.(261155)" 
+    cost_center_id:
+      type: string
+      example: "in.(7475)"
+    currency:
+      type: string
+      example: "in.('USD')"
+    vehicle_type:
+      type: string
+      example: "in.('fourwheeler1')"
+    employee_id:
+      $ref: '#/employee_id'
+
+expense_policy_individual_desired_states:
+  type: object
+  properties:
+    rule:
+      $ref: '#/policy_rule_minimal'
+    amount:
+      allOf:
+        - $ref: '#/amount'
+      example: 500
+    violating_expenses_query_params:
+      $ref: '#/expense_policy_violating_expenses_query_params'
+
+expense_policy_final_desired_state:
+  type: object
+  properties:
+    expense_id:
+      type: string
+      example: "txabsdfSYqd"
+    flag:
+      type: boolean
+      example: True
+    amount:
+      allOf:
+        - $ref: '#/amount'
+      example: 500
+    is_receipt_mandatory:
+      type: boolean
+      example: True
+    add_approver_user_ids:
+      type: array
+      items:
+        type: string
+      example:
+        - ouBdsdfscl
+        - ouLSsdfwqt
+    remove_approver1:
+      type: boolean
+      example: True
+    action_description:
```

Comment:
give offline thought once, that the current API will help the client to build the application without any issue - updating the code in client is acceptable, but they should have all the info available to write the logic